pep-9999.rst

```diff
@@ -202,6 +202,24 @@ No::
     def bar(y: str = None) -> None: ...
     def baz(z: Optional[str] = None) -> None: ...
 
+Do not annotate ``self`` and ``cls`` in method definitions, except when
+referencing a type var.
```

Comment:
Nit: I feel like this should say either "TypeVar" (the name in code) or "type variable" (the unabbreviated name).